internal/move2kube/planner.go

```diff
@@ -69,6 +69,9 @@ func CuratePlan(p plantypes.Plan) plantypes.Plan {
 		cachepaths = append(cachepaths, p.Spec.Inputs.QACaches[i])
 	}
 	qaengine.AddCaches(cachepaths)
+	if len(p.Spec.Inputs.Services) == 0 {
```

Comment:
We have to do the same even if curate option is not selected. This code gets invoked only if we invoke the single command flow or translate -c.

Reply:
done